internal/host/hostcommands/verify_vips_cmd.go

```diff
@@ -0,0 +1,81 @@
+package hostcommands
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/models"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+type verifyVipsCmd struct {
+	baseCmd
+	db *gorm.DB
+}
+
+func newVerifyVipsCmd(log logrus.FieldLogger, db *gorm.DB) CommandGetter {
+	return &verifyVipsCmd{
+		baseCmd: baseCmd{log: log},
+		db:      db,
+	}
+}
+
+func (f *verifyVipsCmd) prepareParam(host *models.Host) (string, error) {
+	var inventory models.Inventory
+	err := json.Unmarshal([]byte(host.Inventory), &inventory)
+	if err != nil {
+		f.log.WithError(err).Warn("Inventory parse")
+		return "", err
+	}
+	if host.ClusterID == nil || *host.ClusterID == "" {
+		return "", nil
+	}
+	cluster, err := common.GetClusterFromDB(common.LoadTableFromDB(common.LoadTableFromDB(f.db, common.APIVIPsTable), common.IngressVIPsTable),
+		*host.ClusterID, common.SkipEagerLoading)
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to get cluster %s", host.ClusterID.String())
+	}
+	if len(cluster.APIVips) == 0 && len(cluster.IngressVips) == 0 {
+		return "", nil
```

Comment:
Shouldn't happen, but worth a log?

Reply:
May happen if VIPs were not defined yet.  Logging will cause noise in he log file.